_docs/developer-information/contribute-to-drill/020-apache-drill-contribution-ideas.md

```diff
@@ -84,8 +77,8 @@ Initially, concentrate on basics:
 
 Implement custom storage plugins for the following non-Hadoop data sources:
 
-  * NoSQL databases (such as Mongo, Cassandra, Couch etc)
-  * Search engines (such as Solr, Lucidworks, Elastic Search etc)
+  * NoSQL databases (such as Mongo, Couch etc)
```

Comment:
Mongo is implemented.... maybe remove?  There actually is a Couchbase plugin out there also that someone implemented.  I've been trying to get them to commit that.

Reply:
what about clickhouse?

Reply:
Sure... Please add to the list!

Reply:
I'd also add better support for streaming data such as Apache Flink and Pulsar.

Reply:
@Leon-WTF Nice idea. then you can create a JIRA about the `Add Storage Plugin for Clickhouse` directly, we assign to you.

Reply:
Clickhouse runs SQL, even if their dialect does also include a good serving of nonstandard stuff, and has a JDBC driver so I think that it should already work to a significant extent...

Reply:
Sorry for the misunderstand, I push two different commits together, and I will roll back the update of apache-drill-contribution-ideas.md, and we could use another PR to update it specificly.